index.html

```diff
@@ -309,6 +311,93 @@ <h1 class="page-header">Red October Management</h1>
 				</form>
 			</div>
 		</section>
+		<hr />
+		<section class="row">
+			<div id="orders" class="col-md-6">
+				<h3>Create Order</h3>
```

Comment:
There's no link in the navbar for this. This entire file also needs to be synced with redoctober.go.